README.md

```diff
@@ -21,6 +21,15 @@ to change the admin password.
 docker logs ovirt-engine
 ```
 
+## Run ovirt-engine 4.0 using kubevirt environemnt [1]:
```

Comment:
The json files are working on top of kubernetes (they are not kubevirt related) and they are just examples.
The repo is about running ovirt-engine in a container.

It would for instance be completely valid to to try to run ovirt-engine on top of kubernetes instead of hosted-engine (although some wrapping of the engine process with etcd leases would be required to achieve that) Hm, we should add an example for that too ...

Further what you have here is just how to deploy it on the kubevirt development environment. That is very specific and it could change any time.